driftbase/api/players/counters.py

```diff
@@ -185,11 +57,27 @@ def get(self, player_id):
         if not get_player(player_id):
             abort(404, message="Player Not found")
```

Comment:
Maybe use http_client.NOT_FOUND instead of literal 404 in abort.

Reply:
Fixed